### ISSUE TITLE
Scaling additional regressors and forecast value?

### ISSUE THREAD

**Issue by pisarevaa** [facebook/prophet#1843]
Hi, everone! First question, should I scale additional regressors before using in fitting Prophet model or Prophet does it on itself? Second question, should I scale additional regressors and also target/forecasting value to the same scale before using in fitting Prophet model or not? I'm using sklearn.preprocessing.StandardScaler library fo scaling.

**Comment by bletham** [comment]
The additional regressors will be standardized by default, but you can turn that off if you want using the `standardize` input:
https://github.com/facebook/prophet/blob/8882c6a3e370736c382db14133f9c66f3b509c30/python/fbprophet/forecaster.py#L588-L594

The target `y` will also be normalized by its maximum (absolute) value:
https://github.com/facebook/prophet/blob/8882c6a3e370736c382db14133f9c66f3b509c30/python/fbprophet/forecaster.py#L350
https://github.com/facebook/prophet/blob/8882c6a3e370736c382db14133f9c66f3b509c30/python/fbprophet/forecaster.py#L327

So by default things will be on the same scale and you shouldn't have to worry about it, but of course if you wanted to try different types of scaling you can disable the regressor standardization as noted above.

**Comment by pisarevaa** [comment]
Thanks a lot!
